app/(afterLogin)/(payroll)/payroll/page.tsx

```
@@ -603,13 +638,14 @@ const Payroll = () => {
       });
       return;
     }
+    console.log('Generating bank letter for amount:', amount);
```

ESLint

[error] Unexpected console statement.

(no-console)